ql/src/test/org/apache/hadoop/hive/ql/session/TestSessionState.java

```diff
@@ -20,21 +20,18 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 
-import java.io.BufferedReader;
 import java.io.File;
 import java.io.IOException;
-import java.io.InputStreamReader;
 import java.lang.reflect.Method;
 import java.util.Arrays;
 import java.util.Collection;
 
 import org.apache.commons.io.FileUtils;
-import org.apache.commons.io.IOUtils;
+import org.apache.hadoop.hive.metastore.Warehouse;
```

Comment:
the package name starts looking a bit too complicated :)
I think we should remove at least hive from the package hierarchy...(followup)

Reply:
I agree hadoop.hive is irritating, but since 95% of Hive classes are like that, it seems better to leave it that way.